content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,365 @@
+
+Community member nodes MUST choose one of the following options:
+
+1. **Download all archives** - Request and download all data pieces for `data` provided by the torrent (this is the case for new community member nodes that haven't downloaded any archives yet)
+2. **Download only the latest archive** - Request and download all pieces starting at the `offset` of the latest `WakuMessageArchiveIndexMetadata` (this the case for any member node that already has downloaded all previous history and is now interested in only the latst archive)
+3. **Download specific archives** - Look into `from` and `to` fields of every `WakuMessageArchiveIndexMetadata` and determine the pieces for archives of a specific time range (can be the case for member nodes that have recently joined the network and are only interested in a subset of the complete history)
+
+# Storing historical messages
+
+When message archives are fetched, community member nodes MUST unwrap the resulting `WakuMessage` instances into `ApplicationMetadataMessage` instances and store them in their local database.
+Community member nodes SHOULD NOT store the wrapped `WakuMessage` messages.
+
+All message within the same time range MUST be replaced with the messages provided by the message history archive.
+
+Community members nodes MUST ignore the expiration state of each archive message.
+
+# Considerations
+
+The following are things to cosider when implementing this specification.
+
+## Bandwidth consumption
+
+Community member nodes will download the latest archive they've received from the archive index, which includes messages from the last seven days. Assuming that community members nodes were online for that time range, they have already downloaded that message data and will now download an archive that contains the same.
+
+This means there's a possibility member nodes will download the same data at least twice.
+
+## Multiple community owners
+
+It is possible for control nodes to export the private key of their owned community and pass it to other users so they become control nodes as well. This means, it's possible for multiple control nodes to exist.
+
+This might conflict with the assumption that the control node serves as a single source of thruth. Multiple control nodes can have different message histories.
+
+Not only will multiple control nodes multiply the amount of archive index messages being distributed to the network, they might also contain different sets of magnet links and their corresponding hashes.
+
+Even if just a single message is missing in one of the histories, the hashes presented in archive indices will look completely different, resulting in the community member node to download the corresponding archive (which might be identical to an archive that was already downloaded, except for that one message).
+
+# Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+# References
```

Comment:
Maybe add references to the original spec here? or the implementation?

Reply:
This is a tough one. There's no original spec, or are you referring to the PRs that lived in status/specs?
Implementation lives in multiple places in status-go, how would you go about linking to that? 
Listing links to lines of code of some commit hash (to ensure it doesn't break with forward changes)?